.github/workflows/upgrade-python-requirements.yml

```diff
@@ -0,0 +1,71 @@
+name: Upgrade Requirements
+
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
Please remove this trigger, which is perhaps helpful for the development of the action but not helpful over the longer term.